chapters/helloworld/index.rst

```diff
@@ -41,7 +51,11 @@ On Linux:
    :align: center
 
 
-With this script you will download the **Fashion-MNIST** training and test data. Once you have the data, Jina will *index* 60,000 images from the training set. And will use random sample images from the test set as *queries*. Jina will then retrieve the relevant results. After around 1 minute, a web page will open and show results like this:
+Overview
+====================
+
+
+With this script you will download the **Fashion-MNIST** training and test data. Once you have the data, Jina will :term:`index<indexing>` 60,000 images from the training set. And will use random images from the test set as *queries*. Jina will then retrieve the relevant results. After around 1 minute, a web page will open and show results like this:
```

Comment:
Should we convert to numbered list? Each sentence is very short and describes one step

Reply:
Done